src/metabase/api/database.clj

```diff
@@ -87,10 +86,10 @@
                                     :write
                                     :none))))
 
-(defn- card-database-supports-nested-queries? [{{database-id :database} :dataset_query, :as _card}]
+(defn- card-database-supports-nested-queries? [{{database-id :database, :as database} :dataset_query, :as _card}]
   (when database-id
     (when-let [driver (driver.u/database->driver database-id)]
-      (driver/supports? driver :nested-queries))))
+      (driver/database-supports? driver :nested-queries database))))
```

Comment:
Unrelated change. Updated to use `database-supports?` since `supports?` is deprecated